azerothcore/azerothcore-wotlk
# fix (core) GOs live add

This fixes gameobjects issue with it being added, not showing on the fly but on restart. Gameobjects now show on the fly when added with gameobject add cmd
![image](https://user-images.githubusercontent.com/16887899/160257882-d18e5cf1-0756-4695-83f1-2d49849f467f.png)



## Changes Proposed:
-  fix gameobjects not spawning on the fly (live) when added with gm cmds

## Issues Addressed:

- Closes Cant locate the report but alot of mentions in discord

## SOURCE:


## Tests Performed:

- builds
- performs


## How to Test the Changes:


1. just do a .gobject add 32349
2. observe you now have a working mailbox at your position
3. do sugar booger

## Known Issues and TODO List:


- [x] sugar booger
- [x] wut?


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
